examples/values/values.go

```diff
@@ -1,6 +1,7 @@
-// Go has various value types including strings,
-// integers, floats, booleans, etc. Here are a few
-// basic examples.
+// Go ha una serie di valori di tipo differente,
```

Comment:
Allora penso che sia *tipi di valori* che *valori di tipo* portino confusione in chi legge.
Si potrebbe cambiare con:
 *In Go è possibile utilizzare valori di svariati tipi
fra i quali anche string, integer, boolean*